packages/cubejs-server-core/src/core/server.ts

```diff
@@ -173,15 +173,13 @@ export class CubejsServerCore {
       schemaPath: process.env.CUBEJS_SCHEMA_PATH || 'schema',
       ...opts,
     };
-
-    if (
```

Comment:
Maybe it's better to pass an additional flag in options to disable this checks when the wizard is in progress? Instead of commenting this code